omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/EncounterController1_9Test.java

```diff
@@ -141,6 +141,68 @@ public void createEncounter_shouldCreateEncounterWithObsAttributesUnordered() th
 	}
 	
 	@Test
+	public void createEncounter_shouldCreateEncounterWithNestedObs() throws Exception {
+		long before = getAllCount();
+		
+		List<SimpleObject> obs = new ArrayList<SimpleObject>();
+		List<SimpleObject> parentGroupMembers = new ArrayList<SimpleObject>();
+		List<SimpleObject> child1GroupMembers = new ArrayList<SimpleObject>();
+		
+		SimpleObject child1child1 = new SimpleObject();
+		child1child1.put("value", 1);
+		child1child1.put("concept", "c607c80f-1ea9-4da3-bb88-6276ce8868ff");
```

Comment:
These concept UUIDs needs to refer to test concept UUIDs, for example numeric one like this: `c607c80f-1ea9-4da3-bb88-6276ce8868dd`. Otherwise REST would not know where should it assign this numeric value (NullPointerException in this case).